geode-redis/src/main/java/org/apache/geode/redis/internal/RedisCommandType.java

```diff
@@ -2595,6 +2598,69 @@ public RedisDataType getDataType() {
     }
   },
 
+  /***************************************
+   ********** Publish Subscribe **********
+   ***************************************/
+  /**
+   * SUBSCRIBE channel...
+   * <p>
+   * subscribe to channel
+   */
+  SUBSCRIBE {
+    private Executor executor;
+
+    @Override
+    public Executor getExecutor() {
+      if (executor == null) {
+        executor = new SubscribeExecutor();
+      }
+      return executor;
+    }
+
+    private final RedisDataType dataType = RedisDataType.REDIS_PUBSUB;
+
+    @Override
+    public RedisDataType getDataType() {
+      return this.dataType;
+    }
+  },
+  PUBLISH {
+    private Executor executor;
+
+    @Override
+    public Executor getExecutor() {
```

Comment:
Not really, but since this pattern is used throughout all the commands I think it would be better to make that change in a separate PR and cover all the commands.

Reply:
Could we please raise one and address accordingly